backend/agentd/session.go

```diff
@@ -242,6 +236,9 @@ func (s *Session) Stop() {
 			logger.Debug(err)
 			break
 		}
+	}
+	for _, sub := range s.cfg.Subscriptions {
+		topic := messaging.SubscriptionTopic(org, env, sub)
```

Comment:
Why not simply remove `break` at line 237?

Reply:
If the message bus shuts down, we don't need to continue unsubscribing, but that's not relevant to the ring.

Reply:
I see, thanks!